.env

```diff
@@ -0,0 +1,2 @@
+PROD_URI=https://movies-api-react-mentoring.herokuapp.com
+DEV_URI=http://localhost:4000
```

Comment:
Security-wise, .env must always be ignored by Git and must not be present in a repo.
You either use .env locally, or keep your secrets separately in the entire different place (in your case - Settings tab of the repo).
Since these secrets are just URLs, it's fine - but do keep the above in mind.